syntax = "proto3";

message SumRequest {
  double a = 1;
  double b = 2;
}

message SumReply {
  double s = 1;
}

service Calculator {
  rpc Sum (SumRequest) returns (SumReply);
  rpc Multiply (MultiplyRequest) returns (MultiplyReply);
}

message MultiplyRequest {
  double a = 1;
  double b = 2;
}

message MultiplyReply {
  double s = 1;
}
